### ISSUE TITLE
Chart missing from index.d.ts

### ISSUE THREAD

**Issue by darewreck54** [jerairrest/react-chartjs-2#268]
I'm trying to register a plugin component and notice that the type definition is missing Chart.  Was this by design? 

Also how do you actually register a plugin component without this definition?

Thanks,
Derek

**Comment by koltclassic** [comment]
@darewreck54 Have you found a workaround for this issue? I've also been attempting to add plugins to my charts, though it looks like both the "mix" plugin example chart and the plugin section of the README are outdated.

**Comment by Hianz** [comment]
Accessing Chart.Chart will work for registering plugins.
```javascript
import ChartJS from "react-chartjs-2"
const Chart: any = ChartJS

    componentWillMount() {
        Chart.Chart.pluginService.register({
            afterDraw: function (chart: any, easing: any) {
                console.log("Plugin registered")
            }
        })
    }
```

**Comment by koltclassic** [comment]
@Hianz Thanks, that works great! Would extending the functionality of existing charts work in a similar way? I see that in the `react-chartjs-2` documentation it is mentioned, but the only example is the plugin example. I attempted to implement extending the Doughnut functionality in a similar way to how you described registering a plugin, following a similar path as is described here: https://www.chartjs.org/docs/latest/developers/charts.html under `Extending Existing Chart Types` . I updated the references to use `Chart.Chart`, but that doesn't seem to have worked properly. I feel like it may be related to the base `<Doughtnut />` component being called in render, though I've passed in my extended chart type into the config as the `type`. I can't seem to access anything inside of `Chart.Chart.controllers.doughnut.extend({...})`. Thank you again for the help!
